### PR TITLE
chore: remove cifar with splitting

### PR DESCRIPTION
**Pull request by andrei-stoian-zama**
And additionally:
- change deployment use case to cifar 2b
- move perror search to cifar 2b

Closes https://github.com/zama-ai/concrete-ml-internal/issues/4314
Closes https://github.com/zama-ai/concrete-ml-internal/issues/3773

### CODE REVIEW

File: use_case_examples/cifar/cifar_brevitas_training/p_error_search.py
Comment (jfrery) [inline use_case_examples/cifar/cifar_brevitas_training/p_error_search.py:1]: Introducing this here probably requires some changes in the readme + have you run it? What is the final `p_error` found? It's likely not going to be 0.1

**Review by jfrery — CHANGES_REQUESTED** [review]
Thanks for tackling this.

File: use_case_examples/cifar/cifar_brevitas_training/p_error_search.py
Comment (andrei-stoian-zama) [reply use_case_examples/cifar/cifar_brevitas_training/p_error_search.py:1]: No, I didn't run it, it's way too long.

We can run it later with the new improved simulation

**Review by jfrery — APPROVED** [review]
Looks good to me. Could you please create the related issue to reintroduce p_error search for cifar with new simulation?

**Pull request merged**